src/System.Linq.Expressions/src/Resources/System.Linq.Expressions.rd.xml

```diff
@@ -6,6 +6,7 @@
       <TypeInstantiation Name="System.Linq.Expressions.ExpressionCreator" Arguments="System.Func{System.Boolean}" Dynamic="Required All" />
       <TypeInstantiation Name="System.Runtime.CompilerServices.CallSite" Arguments="System.Func{System.Runtime.CompilerServices.CallSite, System.Object, System.Int32}" Dynamic="Required All" /> 
       <Type Name="System.Runtime.CompilerServices.CallSiteOps" Dynamic="Required All" />
+      <Type Name="System.Runtime.CompilerServices.RuntimeOps" Dynamic="Required All" />
```

Comment:
Could you please create an issue for the area owners to verify that this is the proper fix? It's kind of a big hammer solution - which works, but it also means that the type will be rooted for all applications, regardless if it's used or not.